internal/expr/parser.go

```diff
@@ -400,36 +421,39 @@ func (p *Parser) skipBlanks() bool {
 func (p *Parser) skipString(s string) bool {
 	if p.pos+len(s) <= len(p.input) &&
 		strings.EqualFold(p.input[p.pos:p.pos+len(s)], s) {
+		// Manually advance the parser to the end of the string.
```

Comment:
Is this comment necessary? I think this function is short enough that its doc-comment is sufficient :P

Reply:
This `skipString` func is funky. We're using that `EqualFold` function because its case insensitive, this is really the only point in the parser where we are not matching the chars ourselves. This means its also the only point where we manually advance the parser outside of `nextChar`.

Tbh I should put some more docs in this func to explain that its a bit strange.